src/shared/db.py

```diff
@@ -219,6 +238,15 @@ def get_user(self, uid: str) -> Optional[SubHubDeletedAccountModel]:
             logger.error("get user", uid=uid)
             return None
 
+    def find_by_uid(self, uid: str):
+        items = [item for item in self.model.query(uid)]
```

Comment:
why the list comprehension?

Reply:
@bowlofstew I believe you wrote this one.

Reply:
it appears as thought it could just `return self.model.query(uid)` and be done with it, no? @bowlofstew

Reply:
This goes back to the cardinality discussion earlier.  If our model can guarantee it is unique then we can simplify but that `query` call technically may be a 1 to many call per the data model.

Reply:
Are you saying `model.query()` can return scalar OR list?  Even so can you explain how a list comprehension deals with whatever you think is happening please?

Reply:
model.query() does return a list.  In the current setup we are not enforcing user_id to be unique, therefore there could potentially be multiple records with the same user_id.  This is preventable through code, obviously.